constantine/math/polynomials/polynomials.nim

```diff
@@ -143,6 +143,32 @@ type InvDiffArrayKind* = enum
   kArrayMinus
   kMinusArray
 
+# How do we do modulo? Field.fromBigInt()?
```

Comment:
Are you talking about polynomial division / remainder or coefficients?

If it's coefficient, it's handled internally when you use fields.

Reply:
@mratsim ,Yes, I wanted to ask about that:
1. How do I perform `n % BLS_MODULUS`?
2. How do I do division between 2 polynomials given in form of their polynomial coefficients? I see that there are methods for sum and product.

Reply:
> 1. How do I perform n % BLS_MODULUS?

Field element do not need this, all operations are implicitly reduced modulo BLS_MODULUS.

> 2. How do I do division between 2 polynomials given in form of their polynomial coefficients? I see that there are methods for sum and product.

Where is it used in the spec? EIP-4844 introduces polynomial division but for polynomial in Lagrange/evaluation form. I would expect the spec to introduce the polynomial division code if it's used.

Ultimately it's similar to integer division because integers are polynomials but with a twist, they carry.